.github/workflows/dissect-ci-template-self-hosted.yml

```diff
@@ -107,44 +67,20 @@ jobs:
           - python-version: "3.12"
             allow_failure: true
     steps:
-      - if: ${{ inputs.run_tests == true && inputs.deb-packages != '' && ! startsWith(matrix.os, 'windows') }}
+      - if: ${{ inputs.deb-packages != '' && ! startsWith(matrix.os, 'windows') }}
         run: sudo apt-get install -qq ${{ inputs.deb-packages }}
-      - if: ${{ inputs.run_tests == true }}
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
 
-      # Caching LFS objects
-      - if: ${{ inputs.run_tests == true }}
-        name: Create LFS file list
-        run: git lfs ls-files --long | cut -d ' ' -f1 | sort > .lfs-assets-id
-      - if: ${{ inputs.run_tests == true }}
-        name: Cache LFS
-        uses: actions/cache@v4
-        with:
-          path: .git/lfs
-          key: lfs-${{ hashFiles('.lfs-assets-id') }}-v1
-      - if: ${{ inputs.run_tests == true }}
-        name: Pull LFS
-        run: git lfs pull
+      #TODO: Before merge, change to @main
+      - uses: fox-it/dissect-workflow-templates/.github/actions/git-checkout@cleanup
 
-      - if: ${{ inputs.run_tests == true }}
-        uses: actions/setup-python@v5
+      - uses: fox-it/dissect-workflow-templates/.github/actions/tox-run@cleanup
         with:
-          python-version: ${{ matrix.python-version }}
-          cache: "pip"
-          cache-dependency-path: "pyproject.toml"
-      - if: ${{ inputs.run_tests == true }}
-        run: pip install "tox==${{ matrix.tox-version }}"
-      - if: ${{ inputs.run_tests == true && github.ref_name == 'main'  && ! startsWith(matrix.os, 'windows') }}
-        run: sed -i 's/\[tool.setuptools_scm\]/\[tool.setuptools_scm\]\nlocal_scheme = "no-local-version"/' pyproject.toml
-      - if: ${{ inputs.run_tests == true }}
-        env:
-          PIP_INDEX_URL: ${{ secrets.DISSECT_PIP_INDEX_URL }}
-          C_INCLUDE_PATH: ${{ env.pythonLocation }}/include/${{ matrix.include-prefix }}${{ matrix.python-version }}
-        run: tox run -e ${{ matrix.python-version }}
-      - if: ${{ inputs.run_tests == true }}
-        uses: codecov/codecov-action@v4
+          tox-environment: "${{ matrix.python-version }}"
```

Comment:
Sorry to be anal about these quotes, but what about these (and another instance of these later)?